### ISSUE TITLE
FKESSLER broken starting in cam6_2_034

### ISSUE THREAD

**Issue by cacraigucar** [ESCOMP/CAM#189]
Variables TT_SLOT1, TT_SLOT2, TT_SLOT3, TT_COSB, TT_CCOSB, and TT_1CCOSB were removed in cam6_2_034.  The FKESSLER test has a fincl list which has these variables.  This is causing FLDLST to error out.   The use_case file is dctest_baro_kessler.xml.

Note that these same variables appear in test_tracer_names in the same use_case.  There may need to be modification for these as well.

@brian-eaton @gold2718  please comment on the correct fix.  Hopefully @brian-eaton can include this fix in his upcoming #188 PR

**Comment by brian-eaton** [comment]
The fix for this was to add '-nadv_tt 6' to CAM's configure options.  I thought John did this with his FV3 commit, but I'm looking at config_component.xml and I see that what was added is '-nadv_tt=6'.  That's not correct syntax.  I'll get this fixed.

**Comment by brian-eaton** [comment]
I just tested this in cam6_2_041 and it worked.  Cheryl, did you test in any CAM tags from cam6_2_039 or later?  I had forgotten that the perl GetOptions module does allow the argument syntax -nadv_tt=6.  So I think everything is fine.  We could change the argument for sake of consistency.  Any opinions on that?

**Comment by cacraigucar** [comment]
This all started because the official prealpha FKESSLER test failed for Chris with the latest CESM alpha tag candidate.  Yes, I tested it in several versions of CAM and it failed starting in cam6_2_034 for me.  I was running "create_test ERP_D_Ln9.f19_f19_mg17.FKESSLER.izumi_nag.cam-outfrq9s".

**Comment by cacraigucar** [comment]
Ah - the error message was different in cam6_2_041 (I thought I'd missed the error about TT_SLOT1, etc. and didn't relook at it before making this post).  There is no helpful error message in my run using cam6_2_041.  @fischer-ncar is using cam6_2_040 in his prealpha runs.  @fischer-ncar, was there a useful error message in your prealpha run?

**Comment by brian-eaton** [comment]
Please point me to a test dir.

**Comment by cacraigucar** [comment]
/scratch/cluster/cacraig/ERP_D_Ln9.f19_f19_mg17.FKESSLER.izumi_nag.cam-outfrq9s.20200817_141911_nb75zh/run is the run that crashed using cam6_2_041.

**Comment by brian-eaton** [comment]
Your atm and cpl log files aren't readable by me.

**Comment by cacraigucar** [comment]
Changed permissions so they can be read by everyone

**Comment by brian-eaton** [comment]
Well, that's what I get for not doing my test in vanilla cam6_2_041.  I ran it in the branch I'm about to commit with fixes for FV initialization.  It turns out these fixes are exactly what's needed here.  The problem is that Peter and I added functionality to allow overwriting the default initialization of non-water tracers (which is what the analytic_ic option provides) from fields in the initial file.  We only tested this with the SE dycore and due to some differences in read_inidat between the SE and FV versions, our implementation in FV needed a few minor tweeks.  That's in my PR#188.  So I'll just add that that PR resolves this issue.
